Framework/PythonInterface/plugins/algorithms/WorkflowAlgorithms/SANSILLAutoProcess.py

```diff
@@ -287,13 +301,18 @@ def PyInit(self):
         self.setPropertyGroup('BeamRadius', 'Options')
         self.setPropertyGroup('WaterCrossSection', 'Options')
 
+        self.declareProperty(FloatArrayProperty('MaxQxy', values=[0]),
```

Comment:
I think we should also have reasonable default values for those 2 properties. They can be both -1, in which case we can calculate MaxQxy from the Qmax of the workspace, and deltaQ being 1/32 ot 1/64th of MaxQxy. Can you check?

Reply:
Also please don't forget to add a new system test for Qxy.

Reply:
The default value is now -1 for MaxQxy and DeltaQ. When the default value is used:
* `MaxQxy = Qmax * sqrt(2) / 2`
* `DeltaQ = MaxQxy / 64` if IQxQyLogBinning is false, `MaxQxy / 10` otherwise

There is also a new system test for I(Qx, Qy) output